notebooks/CondensationBulkAnnotations.R.ipynb

```diff
@@ -61,11 +61,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loading required package: rJava\n",
+      "Loading required package: jpeg\n",
+      "Warning message:\n",
+      "“replacing previous import ‘rJava::tail’ by ‘utils::tail’ when loading ‘romero.gateway’”Warning message:\n",
```

Comment:
These warning should be gone now with https://github.com/ome/rOMERO-gateway/pull/34 being merged.